server/server.go

```diff
@@ -66,7 +80,7 @@ func StartServer() {
 		Str("port", port).
 		Msg("Starting API server")
 
-	srv := http.Server{Addr: port, Handler: chi.ServerBaseContext(serverCtx, r)}
+	srv := http.Server{Addr: port, Handler: contextHandler(serverCtx, r)}
```

Comment:
@evq could you take a look at this change?
Renovate failed to update deprecated chi v4. I've updated it to v5.
There is no `chi.ServerBaseContext` available in v5, so I added this.

Reply:
@evq PTAL

Reply:
seems like based on https://github.com/go-chi/chi/blob/master/CHANGELOG.md#v151-2020-12-06 it was deprecated in favor of `http.Server#BaseContext` - should we just use that instead?

Reply:
thanks. I've pushed the update.